src/unity/paths/operator_accounts_accountId.yml

```diff
@@ -0,0 +1,87 @@
+# Oats is making this incredibly annoying with naming conflicts
+get:
+  operationId: GetAccountById
+  tags:
+    - Accounts
+    - Operator
+  parameters:
+    - $ref: '../../common/parameters/TraceSpan.yml'
+    - in: path
+      name: accountId
+      schema:
+        type: string
+      required: true
+      description: The ID of the account to get.
+  responses:
+    '200':
+      description: the account
+      content:
+        application/json:
+          schema:
+              allOf:
```

Comment:
This is a great solution! Does oats support `allOf`? I know the Elixir OpenAPI generators I looked at a while back don't, which is why I ask.

Reply:
I believe it does but I'm not certain. It looks like it's used on other openAPI cloud routes so 🤞🏽